src/OpenLoco/src/Vehicles/Vehicle1.cpp

```diff
@@ -126,13 +130,167 @@ namespace OpenLoco::Vehicles
         return true;
     }
 
+    // 0x004F8974
+    // TODO: Move to track data
```

Comment:
is that for this PR or a future one?

Reply:
fair enough, our policy at work IRL is we don't allow todo comments as they indicate unfinished work, so that's my automatic reaction. will try to remember this for openloco!

Reply:
I'd like to move a lot of things into a track data structure but its a bit of a pain in c++ to do it a bit at a time. Slightly easier to just group it up into one thing. I want it to be like:
```cpp
TrackData::getMeta(trackId).curveSpeedFactor;
TrackData::getMeta(trackId).constructCost;
// etc
```